lib/recurly/event.js

```diff
@@ -0,0 +1,35 @@
+const debug = require('debug')('recurly:event');
+
+const EVENT_NAMES = [
+  'test'
+];
+
+/**
+ * Event dispatcher
+ *
+ * @param {Object} options
+ * @param {Recurly} options.recurly
+ */
+export class EventDispatcher {
+  constructor ({ recurly }) {
+    this.recurly = recurly;
+  }
+
+  send (options) {
+    const data = this.payload(options);
+    this.recurly.request.queued({ method: 'post', route: '/events', data });
+  }
+
+  /**
+   * Formats an event payload
+   *
+   * @param {Object} options
+   * @param {String} options.name event name
+   */
+  payload ({ name }) {
```

Comment:
Are you planning on using other properties in the future? If not, probably easier to just accept a single string argument instead of passing in the whole object and pulling a single prop.

Reply:
Yes exactly, the payload here will accept more properties as they are introduced. Those will come in an upcoming PR to this branch.